androidClient/app/src/main/java/com/macbitsgoa/comrades/ComradesConstants.java

```diff
@@ -6,4 +6,17 @@
  */
 public final class ComradesConstants {
     public static final String DOWNLOAD_DIRECTORY = "Comrades";
+
+    public final static String MESS_A = "A Mess";
```

Comment:
better take the eateries data from firebase

also it is red _chilli_ :P

Reply:
just used as made for ICEF. Menus can be updated for firebase

Reply:
We plan to do do that in the next version, right now due to a stressed timeline, it would be wise to skip it.